src/components/themed/TransactionListTop.tsx

```diff
@@ -326,7 +326,7 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
         // priority assets for swapping.
         const isPriorityAssetMatch =
           currencyWallet.currencyInfo.pluginId === priorityAsset.pluginId &&
-          currencyWallet.enabledTokenIds.some(enabledTokenId => enabledTokenId === priorityAsset.tokenId)
+          (priorityAsset.tokenId == null || currencyWallet.enabledTokenIds.some(enabledTokenId => enabledTokenId === priorityAsset.tokenId))
```

Comment:
Instead of `Array.some`, try `Array.includes` on the right-Rand side. Much simpler.